app/admin/sponsor.rb

```diff
@@ -1,5 +1,19 @@
 ActiveAdmin.register Sponsor do
 
+  actions :all, except: [:destroy]
+
+  total_requested_sponsorships = -> { Sponsor.pluck(:requested_orphan_count).sum }
+  total_active_sponsorships = -> { Sponsorship.all_active.size }
```

Comment:
Not by default, but there is a way to make it happen: http://stackoverflow.com/a/13140275/2623544. I'll push an update shortly. We have other AA helpers that duplicate HQ helper functionality - can either get rid of them too or wait until the whole of AA goes away.